src/containers/admin/index.tsx

```diff
@@ -0,0 +1,311 @@
+    const res = await contract
+      .connect(state.signer)
+      .setPublicSaleStartTime(publicSaleTimestamp);
+  };
+
+  const handlePause = async () => {
+    const res = await contract.connect(state.signer).pause();
+    const events = await res.wait().events;
+    console.log({ events });
+  };
+  const handleUnpause = async () => {
+    const res = await contract.connect(state.signer).unpause();
+    const events = await res.wait().events;
+    console.log({ events });
+  };
+
+  const handleRelease = async () => {
+    const res = await contract.connect(state.signer).release(releaseAmount);
+    console.log({ res });
+  };
+
+  const handleSetRevealTime = async () => {
+    const res = await contract
+      .connect(state.signer)
+      .setRevealAfterTimestamp(revealTimestamp);
+    console.log({ res });
+  };
+
+  const handleSetProjectURI = async () => {
+    const res = await contract.connect(state.signer).setProjectURI(projectURI);
+    console.log({ res });
+  };
+
+  const handleSetReserveTokens = async () => {
+    const res = await contract
+      .connect(state.signer)
+      .reserveTokens(numberOfReserveTokens);
+    console.log({ res });
+  };
+
+  const handleTransferTokens = async () => {
+    const res = await contract
+      .connect(state.signer)
+      .transferReservedToken(addressToTransfer);
+    console.log({ res });
+  };
+
+  return (
+    <Box pt="12rem" px="8rem" pb="4rem">
+      <Text fontSize="5.6rem" fontWeight="bold">
+        Admin Dashboard
+      </Text>
+      <Text fontSize="1.6rem" color="grey" fontWeight="thin">
+        Contract Address: {contractAddress}
+      </Text>
+      <Box mt="4rem">
+        <Text fontSize="4rem" fontWeight="medium">
+          Base Collection
+        </Text>
+        <Divider />
+        <Box row alignItems="center">
+          <LabelledDateTime
+            label="Public Sale Start Time"
+            set={setPublicSaleTimestamp}
+            data={publicSaleTimestamp}
+          />
+          <Box ml="4rem">
+            <Button
+              disabled={!publicSaleTimestamp || publicSaleTimestamp < 0}
+              onClick={handlePublicSaleTimestampUpdate}
+            >
+              Update
+            </Button>
+          </Box>
+        </Box>
+        <Box row>
+          <Box mr="2rem">
+            <Button bg="primary-red" color="white" onClick={handlePause}>
+              Pause
+            </Button>
+          </Box>
+          <Button onClick={handleUnpause}>Unpause</Button>
+        </Box>
+      </Box>
+      <Box mt="4rem">
+        <Text fontSize="4rem" fontWeight="medium">
+          Payment Splittable
+        </Text>
+        <Divider />
+        <Text fontSize="2rem" fontWeight="bold">
+          Total Earnings: {totalReleased + contractBalance} ETH
+        </Text>
+        <Box row alignItems="center">
+          <LabelledInput
+            label="Amount to Release (in ETH)"
+            set={setReleaseAmount}
```

Comment:
maybe, just a workaround
```suggestion
            set={(amount) => setReleaseAmount(parseUnits(amount, 18))}
```

Reply:
toString(), because, you use releaseAmount as data, and if it is amount, it would give an error, when rendering it.

Reply:
made this change in the `handleRelease()` method as this `<LabelledInput />` component is a custom reusable component and needs to changes to implement here.